vehicle_mass :              1.5       # kg
motor_time_constant:        0.02      # sec
motor_rotational_inertia:   6.62e-6   # kg m^2
thrust_coefficient:         1.91e-6   # N/(rad/s)^2"
torque_coefficient:         2.6e-7    # Nm/(rad/s)^2
drag_coefficient:           0.1       # N/(m/s)
aeromoment_coefficient_xx:  0.003     # Nm/(rad/s)^2
aeromoment_coefficient_yy:  0.003     # Nm/(rad/s)^2
aeromoment_coefficient_zz:  0.003     # Nm/(rad/s)^2
max_prop_speed:             2200      # rad/s
moment_process_noise:       1.25e-7   # (Nm)^2 s
force_process_noise:        0.0005    # N^2 s"

moment_arm:                 0.08      # m ???

accelerometer_biasprocess:  0.0       # m^2/s^5, 1.0e-7
gyroscope_biasprocess:      0.0       # rad^2/s^3, 1.0e-7
accelerometer_biasinitvar:  0.00001   # (m/s^2)^2, 0.005
gyroscope_biasinitvar:      0.00001   # (rad/s)^2, 0.003
accelerometer_variance:     0.0001    # m^2/s^4, 0.001
gyroscope_variance:         0.00001   # rad^2/s^2, 0.001